core/src/main/java/org/apache/iceberg/MetricsConfig.java

```diff
@@ -48,6 +48,14 @@ public static MetricsConfig getDefault() {
     return spec;
   }
 
+  public static MetricsConfig positionDeleteConfig() {
```

Comment:
It is an open question whether we want to persist metrics for the deleted row. During scans, we could probably filter out delete files where delete rows do not match the scan condition.

Reply:
Looking for feedback on this point.

Reply:
Sounds like a good idea to me. I have seen that for Parquet we already push down the filter when reading the delete files. It would be even better if we do not open the file if we do not have to.

Reply:
That's what I am inclined to right now as well.

Reply:
I'll wait for @szehon-ho's PR #2240 to be in and then provide something like:

```
MetricsConfig.forPositionDelete(table)
```

Naming suggestions are welcome as always.

Reply:
Updated as discussed.